ouroboros-network/src/Ouroboros/Network/PeerSelection/PeerSelectionActions.hs

```diff
@@ -185,21 +206,24 @@ withPeerSelectionActions
                               dnsActions
                               ($ n)
 
-    requestBigLedgerPeers
-      :: DNSSemaphore m
-      -> (NumberOfPeers -> LedgerPeersKind -> m (Maybe (Set peeraddr, DiffTime)))
-      -> Int
-      -> m (Set peeraddr, DiffTime)
-    requestBigLedgerPeers dnsSemaphore getLedgerPeers n = do
-      peers_m <- getLedgerPeers (NumberOfPeers $ fromIntegral n) BigLedgerPeers
-      case peers_m of
-        Nothing    ->  do
-          (m, dt) <- requestConfiguredRootPeers dnsSemaphore n
-          -- TODO: we need to ensure we only choose from big configured root
-          -- peers, but for that the root peers need to contain stake
-          -- information!
-          return (Map.keysSet m, dt)
-        Just peers -> return peers
+    requestConfiguredBootstrapPeers :: DNSSemaphore m -> Int -> m (Set peeraddr, DiffTime)
+    requestConfiguredBootstrapPeers dnsSemaphore n = do
+      let readBootstrapPeersMap =
+            fmap (\case
+                    DontUseBootstrapPeers     -> Map.empty
+                    UseBootstrapPeers domains ->
+                      Map.fromList (map (\a -> (RelayDomainAccessPoint a, DoNotAdvertisePeer))
+                                        domains)
+                 )
```

Comment:
@njd42 mentioned that we need to support `ip` addresses in boostrap peers.

Reply:
Right! Forgot about that!